smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/paginated-trait-test.errors

```diff
@@ -18,3 +18,7 @@
 [ERROR] ns.foo#UnresolvedOutput$nextToken: member shape targets an unresolved shape `ns.foo#Missing` | Target
 [ERROR] ns.foo#Invalid9: When bound within the `ns.foo#Service` service, paginated trait `inputToken` is not configured | PaginatedTrait
 [ERROR] ns.foo#Invalid9: When bound within the `ns.foo#Service` service, paginated trait `outputToken` is not configured | PaginatedTrait
+[WARNING] ns.foo#DeeplyNestedOutputOperation: paginated trait `items` should not include a path with more than two parts | PaginatedTrait
```

Comment:
Since this is the reference implementation of all things Smithy, this guidance here comes off almost as giving permission to implementers that more than 2 levels deep isn't supported. Can you try to rephrase this to be more about avoiding more than 2 levels to improve API quality?